src/js-tracker.js

```diff
@@ -198,11 +198,10 @@ function TrackerAsset() {
                 }
 
                 tracker.auth = data.authToken;
-                if(data.actor) {
-                    tracker.actor = new TrackerEvent.TraceActor(data.actor.name, data.actor.account.name, data.actor.account.homePage);
-                } else {
-                    tracker.actor = new TrackerEvent.TraceActor("undefined", "undefined", "undefined");
-                }
+                actor_name = data.actor.name == null ? "undefined" : data.actor.name;
+                account_name = data.actor.account.name == null ? "undefined" : data.actor.account.name;
```

Comment:
actor_name and account_name must never be null neither undefined, at least should be a token "XYZ..". If not available should raise an error.

Reply:
fix

Reply:
Using "XYZ.." was just an example. The best option is to raise an error, but if not feasible, define a "token" that is understandable.

Reply:
raise a message now